.goreleaser.yml

```diff
@@ -58,23 +80,49 @@ builds:
     gcflags: *gcflags
     ldflags: *ldflags
 
+archives:
+  - id: "gnu"
+    builds:
+      - "linux-amd64-gnu"
+      - "linux-arm64-gnu"
+    name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}_gnu"
```

Comment:
nit: we don't have any mips builds

Reply:
This is the default value from the goreleaser docs. I just took the default regex and added our suffix